x-pack/plugins/reporting/server/lib/tasks/execute_report.ts

```diff
@@ -30,6 +30,13 @@ import {
 } from './';
 import { errorLogger } from './error_logger';
 
+interface ReportingExecuteTaskInstance /* extends TaskInstanceWithDeprecatedFields */ {
```

Comment:
Not sure if this comment should be removed to it is here for clarification purposes.

```suggestion
interface ReportingExecuteTaskInstance {
```